Update add-private-registries.adoc (#1169)
PaloAltoNetworks/prisma-cloud-docs: code-security/admin_guide/get-started/connect-your-repositories/add-private-registries.adoc

Integrating Private Registries with Prisma Cloud enables you to build an accurate dependency tree of the packages in your private registries. Using an *Artifactory* for integration ensures you have visibility into packages of your private registry and fix suggestions for vulnerabilities found in indirect packages.
To remediate vulnerabilities found on Private registries see results and fix suggestions on *Projects*. Alternatively, you can make an informed decision by seeing the scan results on *Supply Chain*, where the graph displays the dependency tree based on the integration.

> NOTE: Currently, Prisma Cloud Code Security supports one Artifactory integration for every package manager and is limited to Maven, Gradle, NPM, Yarn, and Pip.

Author: ```suggestion
NOTE: Currently, Prisma Cloud Code Security supports one Artifactory integration, one registry per package manager and is limited to Maven, Gradle, NPM, Yarn, and Pip.
```
